api/cloudprotocol/unitstatus.go

```diff
@@ -157,11 +158,13 @@ type LayerStatus struct {
 
 // ComponentStatus component status.
 type ComponentStatus struct {
-	ComponentID   string     `json:"componentId"`
-	ComponentType string     `json:"componentType"`
-	Version       string     `json:"version"`
-	Status        string     `json:"status"`
-	ErrorInfo     *ErrorInfo `json:"errorInfo,omitempty"`
+	ComponentID   string          `json:"id"`
```

Comment:
May be better keep it as is componentId

Reply:
we did a changes in all changes - id of component or jayer or service into their statuses agreed to use 'id'. not sure that it need to be reverted - it is json representation

Reply:
ok, good.